calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java

```diff
@@ -119,11 +120,12 @@ public void goToRightMenuTaskEventFromWeekView(String name, selectDayOption opti
     if (date != null && date != "") {
       switch (optionDay) {
       case DETAILTIME:
-        evt.scrollBarToGetElement(By.xpath(ELEMENT_EVENT_TASK_DETAIL_DATE_WEEK_VIEW_ONE_DAY.replace("$name", name)
-                                                                                           .replace("$date", date)));
+        //evt.scrollBarToGetElement(By.xpath(ELEMENT_EVENT_TASK_DETAIL_DATE_WEEK_VIEW_ONE_DAY.replace("$name", name)
```

Comment:
if this code is not relevant anymore, you can delete it, we can use the git history to retrieve this code if needed.